galasa-simplatform-application/src/main/java/dev/galasa/simplatform/listener/AccountTransferListener.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2019.
+ */
+package dev.galasa.simplatform.listener;
+
+import java.io.ByteArrayInputStream;
+import java.util.logging.Logger;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import dev.galasa.simplatform.application.Bank;
+import dev.galasa.simplatform.exceptions.AccountNotFoundException;
+import dev.galasa.simplatform.exceptions.InsufficientBalanceException;
+
+/*
+ * 	AccountTransferListener extends from ListenerManager
+ * 	This 'Listener' is a sub-listener called upon from 
+ * 	ListenerManager.
+ */
+
+public class AccountTransferListener extends ListenerManager {
+	
+	/* The manager variable is the instance of ListenerManager which is currently being talked to */
+	/* This allows us to send our response to the listener which then sends it back to our requester */
+	private ListenerManager manager;
+	
+	private String payload = new String();
+	public String sourceAccountNumber;
+	public String targetAccountNumber;
+
+	public Double value;
+	private String sourceOrgBalance;
+	private String targetOrgBalance;
+
+	private Logger log = Logger.getLogger("Simplatform");
```

Comment:
The Logger should be in the super class so the child doesn't need to declare it

Reply:
Complete